addons/mail/static/src/messaging/component/partner_mention_suggestion/partner_mention_suggestion.js

```diff
@@ -0,0 +1,76 @@
+odoo.define('mail.messaging.component.PartnerMentionSuggestion', function (require) {
+'use strict';
+
+const useStore = require('mail.messaging.component_hook.useStore');
+
+const components = {
+    PartnerImStatusIcon: require('mail.messaging.component.PartnerImStatusIcon'),
+};
+
+const { Component } = owl;
+
+class PartnerMentionSuggestion extends Component {
+
+    /**
+     * @override
+     */
+    constructor(...args) {
+        super(...args);
+        useStore(props => {
+            const composer = this.env.entities.Composer.get(props.composerLocalId);
+            return {
+                composer,
+                partner: this.env.entities.Partner.get(props.partnerLocalId),
+                partnerRoot: this.env.messaging.partnerRoot,
+            };
+        });
+    }
+
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+
+    /**
+     * @returns {mail.messaging.entity.Composer}
+     */
+    get composer() {
+        return this.env.entities.Composer.get(this.props.composerLocalId);
+    }
+
+    /**
+     * @returns {mail.messaging.entity.Partner}
+     */
+    get partner() {
+        return this.env.entities.Partner.get(this.props.partnerLocalId);
+    }
+
+    //--------------------------------------------------------------------------
+    // Handlers
+    //--------------------------------------------------------------------------
+
+    /**
+     * @private
+     * @param ev
+     */
+    _onClickMentionItem(ev) {
+        ev.preventDefault();
+        this.trigger('o-partner-mention-suggestion-clicked');
+    }
+
+    _onMouseOverMention(ev) {
+        this.composer.active_id = parseInt(ev.currentTarget.attributes['data-id'].value);
```

Comment:
change in entity should always follow the `update()` process, never ever write in place from components (at least until entity fields are refactored). Reason is that this by-passes the process, and so prevent calling `_update()` on entities.

Reply:
Done